web/wp-content/themes/dctx/index.php

```diff
@@ -1,21 +1,25 @@
 <?php get_header(); ?>
 
 <div id="main">
+	<h1 class="page-title">Blog</h1>
+
 	<div class="container clearfix">
 
 		<div class="main-content">
 
 			<?php if (have_posts()) : ?>
-				<ul class="posts">
+				<div class="posts">
 					<?php while (have_posts()) : the_post(); ?>
```

Comment:
:no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).